.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -159,4 +190,25 @@ kernel_matrix_testing_upload_deps:
       - $LibvirtSSHKeyARM.pub
       - $STACK_DIR
 
-
+kernel_matrix_testing_run_tests:
+  stage: kernel_matrix_testing
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:7c39524b
+  needs: ["kernel_matrix_testing_upload_deps", "kernel_matrix_testing_setup_env"]
+  tags: ["arch:amd64"]
+  variables:
+    AWS_EC2_SSH_KEY_FILE: $CI_PROJECT_DIR/ssh_key
+    GOVERSION: "1.19"
+  before_script:
+    - !reference [.kernel_matrix_testing_new_profile]
+  script:
+    - INSTANCE_IP=$(cat $CI_PROJECT_DIR/stack.outputs | grep $ARCH-instance-ip | cut -d ' ' -f 2)
+    - MICRO_VM_IP=$(cat $CI_PROJECT_DIR/stack.outputs | grep $ARCH-$TAG | cut -d ' ' -f 2)
+    - chmod 600 $AWS_EC2_SSH_KEY_FILE
+    # Without the newline ssh silently fails and moves on to try other auth methods
+    - echo "" >> $AWS_EC2_SSH_KEY_FILE
```

Comment:
Should we do this when creating the file? That way it is ready to be used everywhere.